src/Core/Configurations/RuntimeConfigProvider.cs

```diff
@@ -130,7 +130,7 @@ public bool TryGetLoadedConfig([NotNullWhen(true)] out RuntimeConfig? runtimeCon
             throw new ArgumentException($"'{nameof(configuration)}' cannot be null or empty.", nameof(configuration));
         }
 
-        if (RuntimeConfigLoader.TryParseConfig(
+        if (FileSystemRuntimeConfigLoader.TryParseConfig(
```

Comment:
Should this be calling the instance method rather than the static method from the `_runtimeConfigLoader` field?

Reply:
fixed.